File: api/domain/Model/Entities/Bill.cs
using domain.Model.Contracts.Enumerators;
using System;

namespace domain.Model.Entities
{
    public class Bill
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Observations { get; set; }
        public decimal Amount { get; set; }
        public DateTime ExpirationDate { get; set; }
        public bool Public { get; set; }
        public string Reference { get; set; }
        public BillStatus Status { get; set; }
    }
}

File: web/js/app/entities/Bill.js
class Bill {

  constructor(id, title, amount, expirationDate, obs, isPublic, reference) {

    this._title = title;
    this._amount = amount;
    this._expirationDate = expirationDate;
    this._observations = obs;
    this._status = 'PENDENTE';
    this._id = id;
    this._isPublic = isPublic;
    this._reference = reference;
  }

  get title() {
    return this._title;
  }

  get amount() {
    return this._amount;
  }

  get expirationDate() {
    return this._expirationDate;
  }

  get observations() {
    return this._observations;
  }

  get status() {
    return this._status;
  }

  set status(value) {
    this._status = value;
  }

  get id() {
    return this._id;
  }

  get public() {
    return this._isPublic;
  }

  get reference() {
    return this._reference;
  }
}
